packages/lib-classifier/webpack.dev.js

```diff
@@ -4,8 +4,13 @@ const path = require('path')
 const webpack = require('webpack')
 
 function gitCommit() {
-  const commitHash = execSync('git describe --always').toString('utf8').trim()
-  return commitHash
+  try {
+    const commitHash = execSync('git describe --always').toString('utf8').trim()
```

Comment:
OOI - could this be 
```
const commitHash = process.env.COMMIT_ID || 'Not a git repository'
```

Reply:
webpack's EnvironmentPlugin should take care of that.
https://webpack.js.org/plugins/environment-plugin/#use-case-git-version

As of webpack 5, you're advised not to reference `process.env` directly but instead use `EnvironmentPlugin` or `DefinePlugin` to set the environment.

Reply:
got it - so the `gitCommit()` function should only run when the `process.env.COMMIT_ID` is missing (i.e. the default fallback), https://webpack.js.org/plugins/environment-plugin/#usage-with-default-values

Reply:
You’re right though: if we don’t need the commit hash in local development, then this could be replaced by a plain string.